docs/docs/installation/docker-compose.mdx

```diff
@@ -44,9 +44,9 @@ More on these two approaches after setting up the requirements for either.
 
 ## Requirements
 
-Note that this documentation assumes that you have [Docker](https://www.docker.com),
-[docker-compose](https://docs.docker.com/compose/), and
-[git](https://git-scm.com/) installed.
+Note that this documentation assumes that you have [Docker](https://www.docker.com) and
+[git](https://git-scm.com/) installed. Note also that we used to use `docker-compose` but that
+is on the path to deprecation so we now use `docker compose` instead.
```

Comment:
```suggestion
```

Reply:
This suggestion goes with the one immediately above it, it should be a combined suggestion.  Kind of confusing rn.